sdk/python/requirements/py3.8-ci-requirements.txt

```diff
@@ -1,6 +1,6 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
you should set up a virtualenv with python 3.8 and actually run this command with python 3.8

Reply:
Yup all done, I naively thought updating the local env var would do everything